src/Microsoft.Health.Dicom.Api.UnitTests/Features/Audit/AuditLoggingFilterAttributeTests.cs

```diff
@@ -174,30 +150,26 @@ public void GivenQueryController_WhenExecutedAction_ThenAuditLogShouldNotBeLogge
         var result = new NoContentResult();
 
         var resultExecutedContext = new ResultExecutedContext(
-            new ActionContext(_httpContext, new RouteData(), new ControllerActionDescriptor() { DisplayName = "Executed Query." }),
+            new ActionContext(_httpContext, new RouteData(),
```

Comment:
AAOOLOAAOTOL - why is all this setup repeated so many times? I know you didn't write these tests.